jsapp/scss/colors.scss

```diff
@@ -1,82 +1,82 @@
 // Kobo global palette of colors.
 
 /*
- * The Kobo "gray" scale
- *
- * NOTE: the list consist of colors with same hue (225) and saturation (16).
+ * The Kobo neutral scale
  */
 
 // (accessible) our black, very very rare, only use if necessary for contrast
-$kobo-gray-14: hsl(225deg 16% 14%); // #1e2129
+$kobo-neutral-900: hsl(225deg 17% 14%); // #1e212a
 
 // (accessible) active text, important text, text on background, top navigation
 // background
-$kobo-gray-24: hsl(225deg 16% 24%); // #333847
+$kobo-neutral-800: hsl(225deg 16% 24%); // #333847
 
 // (accessible) default text, inactive text, options text, buttons text,
 // tabs text, notifications background, hover for popups and dropdowns
-$kobo-gray-40: hsl(225deg 16% 40%); // #565e76
+$kobo-neutral-700: hsl(226deg 16% 42%); // #5a627d
 
 // (readable) clickable icons on hover
-$kobo-gray-55: hsl(225deg 16% 55%); // #7a839f
+$kobo-neutral-600: hsl(225deg 16% 58%); // #828ba5
 
 // icons default (clickable), arrows and indicators default
-$kobo-gray-65: hsl(225deg 16% 65%); // #979fb4
+$kobo-neutral-500: hsl(226deg 16% 70%); // #a6acbf
 
 // inactive icons, some decorative icons
-$kobo-gray-85: hsl(225deg 16% 85%); // #d3d6df
+$kobo-neutral-400: hsl(228deg 16% 82%); // #cacdd9
 
 // scroll bars, table lines, dividers, background of clickable icons on hover
 // when also in a highlighted table row
-$kobo-gray-92: hsl(225deg 16% 92%); // #e7e9ee
+$kobo-neutral-300: hsl(227deg 18% 90%); // #e1e3ea
 
 // default light background, icons on hover, sidebar menu, table headings, …
-$kobo-gray-96: hsl(225deg 16% 96%); // #f3f4f6
+$kobo-neutral-200: hsl(228deg 16% 94%); // #edeef2
 
 // table row highlight on hover
-$kobo-gray-98: hsl(225deg 16% 98%); // #f9f9fb
+$kobo-neutral-100: hsl(240deg 20% 98%); // #f9f9fb
 
 // white background
-$kobo-white: hsl(225deg 16% 100%); // #ffffff
+$kobo-white: hsl(0deg 0% 100%); // #ffffff
 
 /*
  * Primary colours & shades
  */
 
-$kobo-dark-blue: hsl(206deg 100% 31%); // #00599d
-$kobo-hover-blue: hsl(213deg 81% 59%); // #408deb
+$kobo-dark-blue: hsl(207deg 77% 32%); // #135991
+$kobo-hover-blue: hsl(207deg 77% 43%); // #1977c2
```

Comment:
Nitpick but I guess I never noticed "hover-someColor" as a color before, but the naming is a bit vague. Should we only use hover-x in conjunction with the mid-x colors? Does this mean we assume we will never have something that is lighter than mid-x use a hover color?

Reply:
I think that these `hover` colors are part of the palette for the simplicity case. We might decide in future that we would rather have some rule like "`hover` color is 10% darker than the idle color". I agree the name is not perfect, we might decide to change it too in future, if Tessa starts using it for other things :)